### PR TITLE
feat(color): add data viz color palette

### PR DESCRIPTION
**Pull request by kumarivijeta**
## Summary

Added data viz color palette subcomponent under foundation color palette component.

## Figma Link 

https://www.figma.com/design/9NrpK3rmbOk0lhlFkEPSaO/Data-Viz-Component-Library?node-id=1365-884&node-type=frame&m=dev

## ADO Story or GitHub Issue Link

https://dev.azure.com/Kyndryl/Shidoka%20-%20Bridge%20Design%20System/_workitems/edit/1931328

## Checklist

- [x] Used Conventional Commit messages as outlined in the contributing guide.
  - [x] Noted breaking changes (if any).
- [ ] Documented/updated all props, events, slots, parts, etc with JSDoc.
  - [x] Ran the `analyze` command to update Storybook docs.
- [ ] Added/updated Stories with controls to cover all variants.
- [ ] Ran `test` locally to address any failures.
- [ ] Added/updated the Figma link for the Story's Design tab.

## Screenshots

image

### CODE REVIEW

File: src/stories/DataVizColors.stories.js [hunk 1]
```diff
@@ -0,0 +1,2451 @@
...
+              style="background-color: var(--kd-color-data-lilac-10)"
+            ></td>
+          </tr>
+        </tbody>
+      </table>
+    `;
+  },
+};
+
+export const Neutral = {
```
Comment (brian-patrick-3) [inline src/stories/DataVizColors.stories.js:695]: Let's include Neutral in the Palette

Comment (kumarivijeta) [reply src/stories/DataVizColors.stories.js]: Updated

**Review by brian-patrick-3 — APPROVED** [review]

**Comment by srpriyankashetty** [comment]
In sequence 01
Observed that color code is 01-10 in figma and preview its 01-01 for the same color
Same must be checked in all other sequence!!

image

**Review by srpriyankashetty — APPROVED** [review]
Verified the Data Viz color with Figma
Verified the Hex value with Figma

**Pull request merged**